meta-mel/fsl-arm/recipes-devtools/tcf-agent/files/0001-tcf-client-Add-LDFLAGS-to-fix-QA-Warning.patch

```diff
@@ -0,0 +1,54 @@
+From c7399b782d91b18556d9b7827e7351a6e6f20923 Mon Sep 17 00:00:00 2001
+From: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
+Date: Mon, 30 Mar 2015 16:11:19 +0530
+Subject: [PATCH] tcf-client: Add LDFLAGS to fix QA Warning
+
+---
+ Makefile     | 10 +++++-----
+ Makefile.inc |  1 +
+ 2 files changed, 6 insertions(+), 5 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 97038d3..579041d 100644
+--- a/Makefile
++++ b/Makefile
+@@ -29,19 +29,19 @@ $(BINDIR)/libtcf$(EXTLIB) : $(OFILES)
+ 	$(RANLIB)
+ 
+ $(BINDIR)/agent$(EXTEXE): $(BINDIR)/main/main$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+-	$(CC) $(CFLAGS) -o $@ $(BINDIR)/main/main$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BINDIR)/main/main$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
+ 
+ $(BINDIR)/client$(EXTEXE): $(BINDIR)/main/main_client$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+-	$(CC) $(CFLAGS) -o $@ $(BINDIR)/main/main_client$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BINDIR)/main/main_client$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
+ 
+ $(BINDIR)/tcflua$(EXTEXE): $(BINDIR)/main/main_lua$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+-	$(CC) $(CFLAGS) $(EXPORT_DYNAMIC) -o $@ $(BINDIR)/main/main_lua$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LUALIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) $(EXPORT_DYNAMIC) -o $@ $(BINDIR)/main/main_lua$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LUALIBS)
+ 
+ $(BINDIR)/tcfreg$(EXTEXE): $(BINDIR)/main/main_reg$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+-	$(CC) $(CFLAGS) -o $@ $(BINDIR)/main/main_reg$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BINDIR)/main/main_reg$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
+ 
+ $(BINDIR)/valueadd$(EXTEXE): $(BINDIR)/main/main_va$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+-	$(CC) $(CFLAGS) -o $@ $(BINDIR)/main/main_va$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
++	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BINDIR)/main/main_va$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
+ 
+ $(BINDIR)/tcflog$(EXTEXE): $(BINDIR)/main/main_log$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB)
+ 	$(CC) $(CFLAGS) -o $@ $(BINDIR)/main/main_log$(EXTOBJ) $(BINDIR)/libtcf$(EXTLIB) $(LIBS)
+diff --git a/Makefile.inc b/Makefile.inc
+index 9587e9a..2f3d08a 100644
+--- a/Makefile.inc
++++ b/Makefile.inc
+@@ -54,6 +54,7 @@ endif
+ 
+ ifeq ($(OPSYS),GNU/Linux)
+   LIBS = -lpthread -lssl -lcrypto -lrt
++  LDFLAGS = -Wl,--hash-style=gnu
```

Comment:
This will override the LDFLAGS we're passing in. That's not the way to do this. Make it obey LDFLAGS as set by us in the environment, don't hardcode flags we want in its buildsystem.